src/python_pybind11/src/StopWatch.hh

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#ifndef IGNITION_MATH_PYTHON__STOPWATCH_HH_
+#define IGNITION_MATH_PYTHON__STOPWATCH_HH_
+
+#include <pybind11/pybind11.h>
+#include <string>
+
+namespace py = pybind11;
+
+namespace ignition
+{
+namespace math
+{
+namespace python
+{
+/// Define a pybind11 wrapper for an ignition::math::Stopwatch
+/**
+ * \param[in] module a pybind11 module to add the definition to
+ * \param[in] typestr name of the type used by Python
+ */
+void defineMathStopwatch(py::module &m, const std::string &typestr);
```

Comment:
Just some style nits for the future, we don't need to block these PRs for this, we can ticket an issue to come back later and update it all.

```suggestion
/// \brief Define a pybind11 wrapper for an ignition::math::Stopwatch
/// \param[in] _module a pybind11 module to add the definition to
/// \param[in] _typestr name of the type used by Python
void defineMathStopwatch(py::module &_module, const std::string &_typestr);
```